tests/backup/backup_helper.go

```diff
@@ -5284,3 +5291,93 @@ func createBackupUntilIncrementalBackup(ctx context.Context, scheduledAppContext
 	}
 	return incrementalBackupName, nil
 }
+
+// StartAllVMsInNamespace starts all the Kubevirt VMs in the given namespace
+func StartAllVMsInNamespace(namespace string, waitForCompletion bool) error {
+	k8sKubevirt := kubevirt.Instance()
+	var wg sync.WaitGroup
+	errors := make([]string, 0)
+	var mutex sync.Mutex
+
+	vms, err := k8sKubevirt.ListVirtualMachines(namespace)
+	if err != nil {
+		return err
+	}
+	for _, vm := range vms.Items {
+		wg.Add(1)
+		go func(vm kubevirtv1.VirtualMachine) {
+			defer wg.Done()
+			err := StartKubevirtVM(vm.Name, namespace, waitForCompletion)
+			if err != nil {
```

Comment:
you can try errGroup if you are expecting an error handling in go routine
https://bostonc.dev/blog/go-errgroup

Reply:
Cannot be used since the go routine is taking an argument.